externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/FlinkSQLEngine.scala

```diff
@@ -125,6 +124,20 @@ object FlinkSQLEngine extends Logging {
     }
   }
 
+  def executeConfigShell(): Unit = {
+    val flinkHome = System.getenv("FLINK_HOME")
+    require(flinkHome != null, s"FLINK_HOME is null")
+    val commands = Seq("/bin/bash", s"$flinkHome/bin/config.sh")
+    val pb = new ProcessBuilder(commands: _*)
+    val process = pb.start()
```

Comment:
`bash config.sh` is different from `source config.sh`

Reply:
A brief example, assuming you have a `test.sh`
```
export TEST=1
```
And `bash test.sh && echo $TEST` will produce nothing, but `source test.sh && echo $TEST` will produce `1`